tooling/template-base/components/CourseLayout.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import { usePageContext } from '../renderer/usePageContext';
+import CourseSidebar from './CourseSidebar';
+
+export default function CourseLayout({
+  title,
+  coursePages
+}: {
+  title?: string;
+  coursePages: Array<JSX.Element>;
+}) {
+  const pageContext = usePageContext();
+  let pageQuery = 1;
+
+  const { search } = pageContext.urlParsed;
+  if (search?.page && parseInt(search.page)) {
+    pageQuery = parseInt(search.page);
+  }
```

Comment:
`parseInt` gets a little wild when called without a radix, we should always specify `parseInt(foo, 10)`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

It would be ideal to avoid calling this twice as well, we can have a `isNaN` check to set it to a sane value (`1`) if provided invalid input.